### ISSUE TITLE
CHROMEOS: KernelCI tast tests can't file local bundled files

### ISSUE THREAD

**Issue by 10ne1** [kernelci/kernelci-core#2201]
The CrOS build_board.sh script [installs just a few binaries](https://github.com/kernelci/kernelci-core/blob/chromeos/config/rootfs/chromiumos/scripts/build_board.sh#L117) in the tast.tgz build artifacts archive, however some tests require more runtime-time available files.

For example, the storage test `storage.WriteZeroCorrectness` fails with errors like:
```
2023-11-20T13:19:23.932050Z                                       Required data file seq_write missing
2023-11-20T13:19:23.932050Z                                       Required data file tbw_probe missing
2023-11-20T13:19:23.932050Z                                       Required data file 16k_read missing
2023-11-20T13:19:23.932050Z                                       Required data file 16k_write missing
2023-11-20T13:19:23.932050Z                                       Required data file 4k_read missing
2023-11-20T13:19:23.932050Z                                       Required data file 4k_write missing
2023-11-20T13:19:23.932050Z                                       Required data file 4k_read_qd4 missing
2023-11-20T13:19:23.932050Z                                       Required data file 4k_write_qd4 missing
2023-11-20T13:19:23.932050Z                                       Required data file surfing missing
2023-11-20T13:19:23.932050Z                                       Required data file suspend_stress missing
```

These files exist under the tast repo `src/platform/tast-tests/src/go.chromium.org/tast-tests/cros/remote/bundles/cros/storage/data`, but are not installed on devices, so the tests will fail to find them.

Basically these tests definitions are not entirely contained in the Go complied source code, part of the tests exists in those files, so we'd need a mechanism to provide them (maybe in tast.tgz) and the tast code to find & run them.

**Comment by nuclearcat** [comment]
Very impressive catch!
